src/Dapr.Actors/Client/ActorProxyOptions.cs

```diff
@@ -34,14 +41,20 @@ public JsonSerializerOptions JsonSerializerOptions
         public string DaprApiToken { get; set; }
 
         /// <summary>
-        /// The constructor
+        /// Gets or sets the HTTP endpoint URI used to communicate with the Dapr sidecar.
         /// </summary>
-        public ActorProxyOptions()
+        /// <remarks>
+        /// The URI endpoint to use for HTTP calls to the Dapr runtime. The default value will be 
+        /// <c>http://127.0.0.1:DAPR_HTTP_PORT</c> where <c>DAPR_HTTP_PORT</c> represents the value of the 
+        /// <c>DAPR_HTTP_PORT</c> environment variable.
+        /// </remarks>
+        /// <value></value>
+        public string HttpEndpoint { get; set; } = GetDefaultHttpEndpoint();
+
+        private static string GetDefaultHttpEndpoint()
         {
-            if(string.IsNullOrWhiteSpace(this.DaprApiToken))
```

Comment:
I'm not sure why this check was here. Inside the constructor this will always be null.